pandas/tseries/offsets.py

```diff
@@ -972,21 +972,25 @@ class BusinessMonthBegin(MonthOffset):
 
 class _CustomBusinessMonth(_CustomMixin, BusinessMixin, MonthOffset):
     """
-    DateOffset subclass representing one custom business month, incrementing
-    between [BEGIN/END] of month dates.
+    DateOffset subclass representing custom business month(s).
+
+    The count increase between %(bound)s of month dates.
 
     Parameters
     ----------
     n : int, default 1
+        The number of months represented.
     normalize : bool, default False
-        Normalize start/end dates to midnight before generating date range
+        Normalize start/end dates to midnight before generating date range.
     weekmask : str, Default 'Mon Tue Wed Thu Fri'
-        weekmask of valid business days, passed to ``numpy.busdaycalendar``
+        Weekmask of valid business days, passed to ``numpy.busdaycalendar``.
     holidays : list
-        list/array of dates to exclude from the set of valid business days,
-        passed to ``numpy.busdaycalendar``
+        List/array of dates to exclude from the set of valid business days,
+        passed to ``numpy.busdaycalendar``.
     calendar : pd.HolidayCalendar or np.busdaycalendar
+        A business day calendar.
     offset : timedelta, default timedelta(0)
+        Applies an offset counted in valid days.
```

Comment:
Also I don't think an offset needs to be restricted to days - can you reword this? cc @jbrockmendel if you have any suggested wording for this parameter

Reply:
Agreed. It seemed a bit strange to apply a millisecond offset to a month but still it is possible. I changed to a more neutral form if you do not have a better wording for now :)